operator/cmd/run/run.go

```diff
@@ -115,28 +123,29 @@ func (s *LabelSelectorValue) Type() string {
 
 func Command() *cobra.Command {
 	var (
-		clusterDomain               string
-		metricsAddr                 string
-		probeAddr                   string
-		pprofAddr                   string
-		enableLeaderElection        bool
-		webhookEnabled              bool
-		configuratorBaseImage       string
-		configuratorTag             string
-		configuratorImagePullPolicy string
-		decommissionWaitInterval    time.Duration
-		metricsTimeout              time.Duration
-		restrictToRedpandaVersion   string
-		namespace                   string
-		additionalControllers       []string
-		operatorMode                bool
-		enableHelmControllers       bool
-		ghostbuster                 bool
-		unbindPVCsAfter             time.Duration
-		unbinderSelector            LabelSelectorValue
-		autoDeletePVCs              bool
-		forceDefluxedMode           bool
-		helmRepositoryURL           string
+		clusterDomain                   string
+		metricsAddr                     string
+		probeAddr                       string
+		pprofAddr                       string
+		enableLeaderElection            bool
+		webhookEnabled                  bool
+		configuratorBaseImage           string
+		configuratorTag                 string
+		configuratorImagePullPolicy     string
+		decommissionWaitInterval        time.Duration
+		metricsTimeout                  time.Duration
+		restrictToRedpandaVersion       string
+		namespace                       string
+		additionalControllers           []string
+		operatorMode                    bool
+		enableHelmControllers           bool
+		ghostbuster                     bool
+		unbindPVCsAfter                 time.Duration
+		unbinderSelector                LabelSelectorValue
+		autoDeletePVCs                  bool
+		forceDefluxedMode               bool
+		helmRepositoryURL               string
+		enableGhostBrokerDecommissioner bool
```

Comment:
renamed this since last review from "enable-statefulset-decommissioner" to "enable-ghost-broker-decommissioner", because it describes it much better. non-ghost-broker actions are turned off anyway (eg. the PVC stuff)